packages/gnosis/src/index.ts

```diff
@@ -9,7 +9,8 @@ function gnosis(options?: GnosisOptions): WalletInit {
     whitelistedDomains = [
       /^https:\/\/gnosis-safe\.io$/,
       /^https:\/\/app\.safe\.global$/,
-      /^https:\/\/safe\.global$/
+      /^https:\/\/safe\.global$/,
+      /^https:\/\/wallet\.ambire\.com(\/.*)?$/
```

Comment:
Thanks for opening this PR.
These are whitelisted domains that can be passed in and defined by dapp devs. We are only going to support Gnosis approved urls at this point as this package was made in collaboration with the gnosis team. Will close this PR for now. Thanks again!